playwright/support/helper.ts

```diff
@@ -66,3 +66,34 @@ export const checkGoldenOutline = async (
   expect(outlineColor).toEqual("rgb(255, 188, 25)");
   expect(outlineStyle).toEqual("solid");
 };
+
+/**
+ * Asserts if an element has event was calledOnce
+ * @param callbackData an array with callback data
+ * @param eventName {string} event name
+ * @example await expectEventWasCalledOnce(messages, "onClick");
+ */
+export const expectEventWasCalledOnce = async (
+  callbackData: string[],
```

Comment:
We can sort out the types for these utils at a later date imo. I wasn't able to find any way of pulling in `SyntheticBaseEvent` which seems to be part of the internals of playwright but we could do something like the following which would allow us to remove the ts ignores:

```
export interface Foo extends SyntheticEvent<HTMLElement> {
  _reactName: string;
}
```

Reply:
we don't need to do that at least here, we can specify which type of event we are passing inside the `onClick`/`onBlur` etc. function. But then we need to somehow be able to pass it and `.push` it to the array